domains/authorization/Authorization.API/API/Models/Affiliation.cs

```diff
@@ -6,14 +6,12 @@ namespace API.Models;
 public class Affiliation
 {
     private Affiliation(
-        Guid id,
         Guid userId,
         Guid organizationId,
         User user,
         Organization organization
     )
     {
-        Id = id;
```

Comment:
No id?

Reply:
Id is combosite key of organization and user